content/standards/360giving/index.html.md

```diff
@@ -1,23 +1,29 @@
 ---
-title: 360Giving Schema
-category: Domain Specific
-topic:	5.7 Grants, Loans & Subsidies
-subject: Payments Clearing and Settlement
-organisation: threesixtygiving
 name: 360Giving Schema
-status: review
-dateAdded: 2021-02-02
-dateUpdated: 2021-02-02
-classification: Domain Specific
+contraction: 360GS
+specification:
+  name: 360Giving
+  url: https://standard.threesixtygiving.org/en/latest/reference/#spreadsheet-format
+  published: "2018-06-01"
+license_id: CC-BY-4.0
+maintainer: threesixtygiving
+catalogue_status_events:
```

Comment:
could we rename to `status_events`? I'd like to minimise the use of the term 'catalogue'.

Reply:
We had an offline discussion about this, and thought that in this case, it would make it clearer to the contributors that this is _catalogue_ specific status events, opposed to the full DSA workflow